src/deb/control/preinst

```diff
@@ -1,13 +1,20 @@
-#!/bin/bash
-set -o errexit
+#!/usr/bin/env bash
 
-echo 'preinst script executing'
+#set -x
+#set -o errexit
 
-# stop ned service, if exists.    
-if service --status-all 2>&1 | grep -wq 'ned'; then    
-  echo 'sudo service ned stop'
-  sudo service ned stop 
+. /usr/share/debconf/confmodule
 
-  echo 'deleting expanded ned webapps folder'
-  sudo rm -rf /opt/plos/ned/webapps/v*
-fi
+INPUT="db_input high [[artifactId]]"
+
+$INPUT/ned_db_host || true
+$INPUT/ned_db_user || true
+$INPUT/ned_db_password || true
+$INPUT/ringgold_db_host || true
+$INPUT/ringgold_db_user || true
+$INPUT/ringgold_db_password || true
+$INPUT/ringgold_import_dir || true
+$INPUT/http_port || true
+$INPUT/control_port || true
+
+db_go || true
```

Comment:
Ideally please follow the skeleton format generated by dh_make (there should be install, upgrade, and abort-upgrade actions) for preinst; you may not want to run any of this in the case of `abort-upgrade`, for example.